composer.json

```diff
@@ -10,7 +10,7 @@
 		"issues": "https://core.trac.wordpress.org/"
 	},
 	"require": {
-		"php": ">=5.6"
+		"php": ">=7.0"
```

Comment:
Worth doing a composer update?

Reply:
The composer.lock file isn't committed to the repo. What am I missing?